pkgs/by-name/un/unimatrix/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, git
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "unimatrix";
+  version = "unstable-2023-04-25";
+
+  src = fetchFromGitHub {
+    owner = "will8211";
+    repo = pname;
+    rev = "65793c237553bf657af2f2248d2a2dc84169f5c4";
+    hash = "sha256-MtHsoZvUpYPQ5O9CJC7TwcOQOT4ATfyNpoozWYSlI7Q=";
+    # Needed for setuptools-scm to detect version
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [
+    python3Packages.setuptools-scm
+    git
+  ];
```

Comment:
```suggestion
  format = "other";

  src = fetchFromGitHub {
    owner = "will8211";
    repo = pname;
    rev = "65793c237553bf657af2f2248d2a2dc84169f5c4";
    hash = "sha256-MtHsoZvUpYPQ5O9CJC7TwcOQOT4ATfyNpoozWYSlI7Q=";
  };

 installPhase = ''
    runHook preInstall

    install -Dm755 ./unimatrix.py $out/bin/unimatrix

    runHook postInstall
  '';

  dontConfigure = true;
  dontBuild = true;
```

Reply:
I have committed and squashed your changes